src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetObjectForNativeVariantTests.cs

```diff
@@ -51,133 +56,1059 @@ internal struct TypeUnion
         }
 
         [StructLayout(LayoutKind.Explicit)]
-        internal struct Variant
+        public struct Variant
         {
             [FieldOffset(0)] public TypeUnion m_Variant;
             [FieldOffset(0)] public decimal m_decimal;
+
+            public override string ToString() => "0x" + m_Variant.vt.ToString("X");
         }
 
-#pragma warning disable 618
+        public const ushort VT_EMPTY = 0;
```

Comment:
Please do me a favor and mention the Windows header file for reference. `// Taken from wtypes.h` <- I think that is right?

Reply:
Fixed